ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -23,11 +24,18 @@ def initialize(opts = {})
       @map_fail_uri     = opts.fetch(:map_fail_uri, nil)
       @pun_stage_cmd    = opts.fetch(:pun_stage_cmd, "sudo /opt/ood/nginx_stage/sbin/nginx_stage")
 
+      Dotenv.load('/etc/os-release')
+      if (ENV['ID'] == 'rhel' || ENV['ID_LIKE'] =~ /rhel/) && ENV['VERSION_ID'] =~ /^8/
```

Comment:
Dotenv makes it really clean and pithy here. 
That said, if you want to omit the need for the dependency you could replicate the version check used in the update_ood_portal bash script.

```ruby
`[ -f /etc/os-release ] && source /etc/os-release && [[ "$ID $ID_LIKE" = *"rhel"* ]] && [[ "$VERSION_ID" = "8"* ]]`
if $?.success?
  default_htpasswd = "/etc/httpd/.htpasswd"
else
  default_htpasswd = "/opt/rh/httpd24/root/etc/httpd/.htpasswd"
end
```